src/components/AvatarLightbox.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Lightbox from 'react-image-lightbox';
+import getImage from '../helpers/getImage';
+import Avatar from './Avatar';
+
+export default class AvatarLightbox extends React.Component {
+  static propTypes = {
+    username: PropTypes.string,
+    size: PropTypes.number,
+    previewSize: PropTypes.number,
+  };
+
+  static defaultProps = {
+    username: undefined,
```

Comment:
Would it be better to use the default avatar username here? I guess it doesn't matter if it fails on that request then it goes to avatar default image anyways. So your call :)

Reply:
`@steemconnect` uses different avatar than we use as error fallback. I don't know whether there is username that has avatar set to that default avatar.

Reply:
Wrong username got default avatar example https://img.busy.org/@xxxx123456 . We should not rely on @steemconnect avatar, at some point we will want to have the logo of SC as avatar.